ui/components/ConfirmationModal.js

```diff
@@ -62,7 +62,7 @@ const styles = (theme) => ({
     // minWidth : 300,
     padding : theme.spacing(1),
     color : '#fff',
-    backgroundColor : 'rgb(57, 102, 121)',
+    backgroundColor : theme.palette.type == "light"? 'rgb(57, 102, 121)': 'none',
```

Comment:
Hey @ciradu2204 in dark mode, title has a dark color you can see in the below image so if you can change that `none` to the dark color that we are using in this dialog

![Screenshot 2023-05-17 at 10 27 33 PM](https://github.com/meshery/meshery/assets/71271069/b831bd2c-828a-4226-8455-767e8476fe74)